content/ui-patterns/saving.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Saving
+---
+
+GitHub's save patterns help developers to store and update their configurations for different features throughout GitHub's various interfaces. These changes should be represented in the UI accurately, quickly, and obviously. Their behavior should be easy to reason about and inspire confidence and trust.
```

Comment:
developer or users? 🤔

Reply:
I thought developers was a nicer term than a generic "users" but that may not be the most correct. I'm open to changing.

Reply:
Updated.